assets/js/main.js

```diff
@@ -0,0 +1,20 @@
+function convertPokemonToLi(pokemon) {
+    return  `
+        <li class="pokemon ${pokemon.type}">
+            <span class="number">#${pokemon.number}</span>
+            <span class="name">${pokemon.name}</span>
+            <div class="details">
+                <ol class="types">
+                    ${pokemon.types.map(type => `<li class="type">${type}</li>`).join('')}

```

Comment:
What do you think if we extract that part to a `const typelist` in the same function before the return?

Reply:
Good idea, I thought it came out a bit "ugly", but also did not think about how to make it better to read, that's a good solution, thank you!